languages/python/exercises/concept/elyses-enchantments/.docs/introduction.md

```diff
@@ -0,0 +1,242 @@
+["Guava"]
+
+>>> elements_separated_with_commas = ["Parrot", "Bird", 334782]
+["Parrot", "Bird", 334782]
+```
+
+For readability, line breaks can be used when there are many elements or nested data structures within a list:
+
+```python
+>>> lots_of_entries =[
+                        "Rose",
+                        "Sunflower",
+                        "Poppy",
+                        "Pansy",
+                        "Tulip",
+                        "Fuchsia",
+                        "Cyclamen",
+                        "Lavender",
+                        "Daisy",
+                        "Jasmine",
+                        "Hydrangea",
+                        "Hyacinth",
+                        "Peony",
+                        "Dahlia",
+                        "Dandelion",
+                        "Tuberose",
+                        "Ranunculus"
+                      ]
+['Rose', 'Sunflower', 'Poppy', 'Pansy', 'Tulip', 'Fuchsia', 'Cyclamen', 'Lavender', 'Daisy', 'Jasmine', 'Hydrangea', 'Hyacinth', 'Peony', 'Dahlia', 'Dandelion', 'Tuberose', 'Ranunculus']
+
+>>> nested_data_structures = [
+                                 {"fish": "gold", "monkey": "brown", "parrot" : "grey"},
+                                 ("fish", "mammal", "bird"),
+                                 ['water', 'jungle', 'sky']
+                             ]
+[{"fish": "gold", "monkey": "brown", "parrot" : "grey"}, ("fish", "mammal", "bird"), ['water', 'jungle', 'sky']]
+```
+
+The `list()` constructor can be used empty or with an _iterable_ as an argument. Elements in the iterable are cycled through by the constructor and added to the list in order:
+
+```python
+>>> no_elements = list()
+[]
+
+#the tuple is unpacked and each element is added
+>>> multiple_elements_tuple = list(("Parrot", "Bird", 334782))
+["Parrot", "Bird", 334782]
+
+#the set is unpacked and each element is added
+>>> multiple_elements_set = list({2, 3, 5, 7, 11})
+[2,3,5,7,11]
+```
+
+Results when using a list constructor with a string or a dict may be surprising:
+
+````python
+
+#string elements (Unicode code points) are iterated through and added *individually*
+>>> multiple_elements_string = list("Timbuktu")
+['T', 'i', 'm', 'b', 'u', 'k', 't', 'u']
+
+
+>>> multiple_code_points_string = list('अभ्यास')
+['अ', 'भ', '्', 'य', 'ा', 'स']
+
+"""
+The iteration default for dictionaries is over the keys.
+"""
+source_data = {"fish": "gold", "monkey": "brown"}
+>>> multiple_elements_dict_1 = list(source_data)
+['fish', 'monkey']
+
+Because the constructor will only take _iterables_ (or nothing) as arguments, objects that are _not_ iterable will throw a type error. Consequently, it is much easier to create a one-item list via the literal method.
+
+```python
+
+>>> one_element = list(16)
+Traceback (most recent call last):
+   File "<stdin>", line 1, in <module>
+   TypeError: 'int' object is not iterable
+
+>>> one_element_from_iterable = list((16,))
+[16]
+````
+
+## Accessing elements
```

Comment:
I didn't realize that this exercise used slicing in its solution when I wrote this intro (apologies!), so I think that we'll need to add some explanation for _slice notation_ (`[start:stop:step]`) , as well as some explanation of what a slice is (_a copy of a portion of the original list_).

Reply:
I added the explanation of the slice notation in the  intro and about file, I tried my best to make it clear but I don't know if my writing style is adapted to the style of the file. Can you please check that?
Thank you very very much for all the feedback!

Reply:
I added a few small things.  Mostly around trying to clarify slicing - which I think is hard for people to understand on first pass.  I am not so sure I "improved" things, so feel free to reject my suggestions!

Reply:
I find your comments on the code very helpful added those from about to the into doc